.github/workflows/vllm-tensorizer.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     with:
       image-name: vllm-tensorizer
       folder: vllm-tensorizer
-      tag-suffix: ${{ inputs.commit || '19307ba71ddeb7e1cc6aec3c1baa8b50d59c1beb'}}
+      tag-suffix: ${{ inputs.commit || 'b6553be1bc75f046b00046a4ad7576364d03c835'}}
```

Comment:
There shouldn't be a hardcoded commit here... this seems rather odd.

Reply:
It's a bit of a side-effect of not being able to declare variables very elegantly in GHA workflows. Normally this would go well in e.g. a build matrix declaration, but this currently only does one build rather than multiple.

It would also normally work better in a Dockerfile, but then the vLLM commit info wouldn't be accessible to the GHA workflow to add to the image tag.